src/igvfd/searches/configs/measurement_set.py

```diff
@@ -10,9 +10,36 @@ def measurement_set():
             'donors.taxa': {
                 'title': 'Taxa',
             },
+            'samples.classification': {
+                'title': 'Classification',
+            },
+            'samples.sample_terms.term_name': {
+                'title': 'Sample Term',
+            },
+            'samples.targeted_sample_term.term_name': {
+                'title': 'Targeted Sample Term',
+            },
+            'samples.disease_terms.term_name': {
+                'title': 'Disease Term',
+            },
             'assay_term.term_name': {
                 'title': 'Assay Term'
             },
+            'readout.term_name': {
+                'title': 'Readout'
+            },
+            'preferred_assay_title': {
+                'title': 'Preferred Assay Title'
+            },
+            'samples.modifications.modality': {
+                'title': 'CRISPR Modality'
+            },
+            'library_construction_platform.term_name': {
```

Comment:
I think this should be "Library Platform"

Reply:
changed